### PR TITLE
4 otimizar o código detectpy

### PR DESCRIPTION
**Pull request by cauansousa**

### CODE REVIEW

File: src/detect.py [hunk 1]
```diff
@@ -1,32 +1,23 @@
...
 from ultralytics import YOLO
 from ultralytics.yolo.v8.detect.predict import DetectionPredictor
 
 import processing as pr
-#from PIL import Image
 
 class Detector:
 
-    def __init__(self):
+    def _init_(self):
```
Comment (khaledhazime) [inline src/detect.py:20]: This _init_ should not work as expected

File: src/detect.py [hunk 2]
```diff
@@ -60,104 +51,30 @@ def image_callback(self, image):
...
+                    #Load model
                     yolo = YOLO("yolov8n.pt")
                     results = yolo.predict(source=small_frame, conf=0.8)
                     boxes = results[0].boxes
+                    
+                    #Plot bbox 
                     small_frame = pr.plot_bboxes(small_frame, boxes.boxes, conf=0.5)
-
-                    #marked_image = np.squeeze(boxes)
-                    for r in results:
```
Comment (khaledhazime) [inline src/detect.py:87]: Should this be removed?

**Review by khaledhazime — CHANGES_REQUESTED** [review]
I don't think this will work properly. There are some python problems on the code that should be fixed before merging

File: src/detect.py (hunk 2 above)
Comment (cauansousa) [reply src/detect.py:87]: Yes, was used when we were ploting bbox with cv2.

**Review by khaledhazime — APPROVED** [review]

**Pull request merged**